src/components/AttentionBox/__stories__/attentionBox.stories.mdx

```diff
@@ -0,0 +1,162 @@
+  "Their height is based on the content length, which should not exceed two lines of text.",
+]}/>
+
+<Tip title="Check yourself">If the information about a component isn’t critical for the user, use a <Link href="/?path=/docs/components-tooltip--overview" size={Link.sizes.SMALL} withoutSpacing>Tooltip</Link> instead.</Tip>
+
+## Variants
+### States
+There are 4 types of attention boxes: Primary, success, danger, and dark (natural).
+<Canvas>
+  <Story name="States">
+    <MultipleStoryElementsWrapper className="monday-style-attention-box_column-wrapper">
+      <AttentionBox title="Primary" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." componentClassName="monday-style-attention-box_box" />
+      <AttentionBox title="Success" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." type={ATTENTION_BOX_TYPES.SUCCESS} componentClassName="monday-style-attention-box_box" />
+      <AttentionBox title="Danger" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." type={ATTENTION_BOX_TYPES.DANGER} componentClassName="monday-style-attention-box_box" />
+      <AttentionBox title="Dark" text="Lorem Ipsum is simply dummy text of the printing and typesetting industry." type={ATTENTION_BOX_TYPES.DARK} componentClassName="monday-style-attention-box_box" />
+    </MultipleStoryElementsWrapper>
+</Story>
+</Canvas>
+
+### Dismissable 
+<Canvas>
+  <Story name="Dismissable ">
+    <div className="monday-style-attention-box_row-wrapper">
+      <AttentionBox title="Regular attention box" text="Dismissable attention box with two lines of content." onClose componentClassName="monday-style-attention-box_box" />
+      <AttentionBox text="Attention box in compact mode" onClose compact componentClassName="monday-style-attention-box_box" />
+    </div>
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component: <AttentionBox title="Attention box title" text="Studies show that 100% of people who celebrate birthdays, will eventually die." componentClassName="monday-style-attention-box_box"/>,
+        description: "Provide a short and brief title, and explanation related to the title."
+      },
+      negative: {
+        component: <AttentionBox title="Attention box title" />,
+        description: "Don’t use only a title to explain something."
+      }
+    },
+    {
+      positive: {
+        component:
+          <div>
+            <span className="monday-style-attention-box_title">Inbox</span>
+            <span className="monday-style-attention-box_text" >Catch up on updates from all your boards.</span>
+            <AttentionBox text={<span className="monday-style-attention-box_info">Get your monday.com notifications<Link>Learn More</Link></span>} componentClassName="monday-style-attention-box_att-box"  />
+          </div>,
```

Comment:
componentClassName is deprecated for this component, please use className instead.